autogenerate_docs.sh

```diff
@@ -0,0 +1 @@
+pydoc-markdown > autogenerated_docs.md
```

Comment:
There's also the option here of just keeping the output in stdout & copying from there -- I thought this was cleaner, but let me know if you'd prefer to not redirect output to a different file?

Reply:
This is good like this